### ISSUE TITLE
[BUG][0.12.1] Unable to install on `mac10.13`

### ISSUE THREAD

**Issue by roblav96** [microsoft/playwright#1535]
**Context:**
- Playwright Version: v0.12.1
- Operating System: Mac
- Extra: macOS High Sierra 10.13.6

**Code Snippet**
`npm install playwright-core playwright-chromium`

```log
~/project on  master [!?] 
$ npm install playwright-core playwright-chromium

> playwright-chromium@0.12.1 install /Users/admin/project/node_modules/playwright-chromium
> node install.js

(node:54634) UnhandledPromiseRejectionWarning: TypeError: path.join is not iterable (cannot read property Symbol(Symbol.iterator))
    at Object.executablePath (/Users/admin/project/node_modules/playwright-core/lib/server/browserFetcher.js:125:17)
    at downloadOptionsFromENV (/Users/admin/project/node_modules/playwright-core/download-browser.js:44:36)
    at /Users/admin/project/node_modules/playwright-chromium/install.js:21:40
    at Object.<anonymous> (/Users/admin/project/node_modules/playwright-chromium/install.js:22:3)
    at Module._compile (internal/modules/cjs/loader.js:1147:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1167:10)
    at Module.load (internal/modules/cjs/loader.js:996:32)
    at Function.Module._load (internal/modules/cjs/loader.js:896:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47
```

**Describe the bug**

Can't `npm install playwright-chromium`, throws at `lib/server/browserFetcher.js:125:17<executablePath>` due to **`mac10.13`** not being mapped on the `RELATIVE_EXECUTABLE_PATHS`.

**Comment by mxschmitt** [comment]
Which Node.js version are you using?

**Comment by roblav96** [comment]
@mxschmitt 

```
$ node -v
v13.11.0
```
```
$ npm -v
6.14.3
```

**Comment by roblav96** [comment]
The `sw_vers -productVersion` resulting in `mac10.13` need to be mapped on the variables [**`DOWNLOAD_URLS`**](https://github.com/microsoft/playwright/blob/master/src/server/browserFetcher.ts#L40) and [**`RELATIVE_EXECUTABLE_PATHS`**](https://github.com/microsoft/playwright/blob/master/src/server/browserFetcher.ts#L64)

**Comment by mxschmitt** [comment]
In theory we could just fire up a Pull Request to add the mapping there. But not sure if Playwright will support Mac OS 10.13 (was released in September 2017). 
Also the error handling could be improved if an unsupported OS/Version will be used.

cc @JoelEinbinder

**Comment by dgozman** [comment]
We are currently only building bundling WebKit and Firefox for 10.14 and 10.15. @aslushnikov, do you think 10.13 is viable?

**Comment by aslushnikov** [comment]
- I don't think we can build WebKit for 10.13.
- Firefox and Chromium might work on 10.13, but we don't have a 10.13 mac to check it. If someone with 10.13 can try #1549 and see if our downloaded FIrefox / Chromium binaries work, it'll be very helpful!

**Comment by roblav96** [comment]
> We are currently only building bundling WebKit and Firefox for 10.14 and 10.15. @aslushnikov, do you think 10.13 is viable?

I believe `10.13` is still viable as long as `homebrew` continues to publish a `:high_sierra` bottle for the [node](https://github.com/Homebrew/homebrew-core/blob/master/Formula/node.rb#L8) formula.

**Comment by aslushnikov** [comment]
Fixed with #1549
